src/sentry/tasks/check_auth.py

```diff
@@ -46,7 +48,7 @@ def check_auth(chunk_size=100, **kwargs):
             AuthIdentity.objects.filter(id__in=identity_ids_list).update(last_synced=now)
         check_auth_identities.apply_async(
             kwargs={"auth_identity_ids": identity_ids_list, "chunk_size": chunk_size},
-            expires=AUTH_CHECK_INTERVAL,
+            expires=AUTH_CHECK_INTERVAL - AUTH_CHECK_SKEW,
```

Comment:
Why do we need to change how long rabbit messages are valid for?

Reply:
The intent was to make sure that we account for the minimum cutoff in the next runs (where `randrange(AUTH_CHECK_SKEW)` returns `AUTH_CHECK_SKEW`). A 22-24 hour expiration for these jobs feels pretty excessive in general though